src/docfx/config/ops/OpsConfigLoader.cs

```diff
@@ -12,9 +12,10 @@ namespace Microsoft.Docs.Build
 {
     internal static class OpsConfigLoader
     {
-        public static OpsConfig? LoadOpsConfig(ErrorBuilder errors, Package package, string workingDirectory = ".")
+        public static OpsConfig? LoadOpsConfig(
```

Comment:
>LoadOpsConfig [](start = 33, length = 13)

This is still sort of confusing when calling. The caller needs to know if it's from op/docset level.

Reply:
Can we change `workingDirectory` to `relativePathToRepoRoot`?
@yufeih

Reply:
How about passing `Repository?` instead since `.ops.config` is defined as relative to repository root.